source/read/distinct.txt

```diff
@@ -0,0 +1,141 @@
+.. _pymongo-distinct:
+
+==============================
+Retrieve Distinct Field Values
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: read, unique, code example
+
+Overview
+--------
+
+In this guide, you can learn how to retrieve a list of distinct values across a collection for a
+specified field with {+driver-short+}.
```

Comment:
Thought: Would this be easier to parse as multiple sentences? Something like:

```suggestion
Within a collection, different documents might contain different values for a single field. 
For example, one ``restaurant`` document has a ``borough`` value of ``"Manhattan"``, and
another has a ``borough`` of ``"Queens"``.

With {+driver-short+}, you can retrieve all the distinct values
that a field contains across multiple documents.
```

If you like this format, you could do something similar in the subsections, like "We need to find all Italian restaurants in Manhattan" or something

Reply:
Yeah I like that, I was struggling to find a good way to phrase it in a short concise sentence. I changed the overview section to this and left the subsections as-is